docs/git-workflow.md

```diff
@@ -15,7 +15,7 @@ Branches use the following naming conventions:
 
 For example, you can run: `git checkout master` and then `git checkout -b fix/whatsits` to create a new `fix/whatsits` branch off of `origin/master`.
 
-*warning*: Do not use two slashes in the name of a branch. It can too-easily [cause issues](https://stackoverflow.com/questions/2527355/using-the-slash-character-in-git-branch-name).
+**warning**: Do not use two slashes in the name of a branch. It can too-easily [cause issues](https://stackoverflow.com/questions/2527355/using-the-slash-character-in-git-branch-name).
```

Comment:
A minor nitpick is that "cause issues" feels incomplete and "can too-easily" sounds not scary enough 😉.

I might just rephrase the last sentence to be:

>Using multiple slashes has caused problems with Calypso Live testing. See: [Git branches with multiple slashes](https://stackoverflow.com/questions/2527355/using-the-slash-character-in-git-branch-name).

Reply:
It can also cause pretty annoying issues on a local filesystem!